install/ansible/tasks/build.yaml

```diff
@@ -0,0 +1,47 @@
+---
+- name: "Create a directory to download binaries"
+  file: path={{ cri_release_directory }} state=directory
+
+- name: "Get Containerd and CRI-Containerd"
+  get_url:
+    validate_certs: "no"
+    url: "https://storage.googleapis.com/cri-containerd-staging/cri-containerd-{{ cri_containerd_release_version }}.tar.gz"
+    dest: "{{ cri_release_directory }}"
+    mode: 0755
+
+- name: "Unpack Containerd and CRI-Containerd"
+  unarchive:
+    src: "{{ cri_release_directory }}cri-containerd-{{ cri_containerd_release_version }}.tar.gz"
```

Comment:
copy: false

Reply:
I havent been really sure about copy:false and remote_src when I have used this. I will probably target it for a second round optimization ? 
I know that remote_src is usually used when the src is on a remote server but I want to optimize all these tasks into 1 task in the next iteration instead of pulling , unpacking and copying